src/CommunityToolkit.Maui/Converters/ICommunityToolkitValueConverter.shared.cs

```diff
@@ -5,6 +5,16 @@ namespace CommunityToolkit.Maui.Converters;
 /// <inheritdoc />
 public interface ICommunityToolkitValueConverter : IValueConverter
 {
+	/// <summary>
+	/// Type Converting From
+	/// </summary>
+	Type FromType { get; }
```

Comment:
I suspect this is here for symmetry but do we need it?

Reply:
Yup, exactly 💯.

It doesn't affect the bug in #410, but I figured it'd be good to add it for symmetry.